apps/src/studio/studio.js

```diff
@@ -1066,7 +1066,9 @@ Studio.onTick = function () {
     Studio.executeQueue('when-right');
     Studio.executeQueue('when-down');
 
-    Studio.executeQueue('askCallback');
+    for (let i = 0; i < Studio.askCallbackIndex; i++) {
+      Studio.executeQueue(`askCallback${i}`);
```

Comment:
I'd like to repeat my request for a comment here explaining the purpose of this queue, particularly given that it is now _more_ complex